IISIntegration.NoV1.sln

```diff
@@ -113,6 +113,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IIS.Tests", "test\IIS.Tests
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Common.Tests", "test\Common.Tests\Common.Tests.csproj", "{D17B7B35-5361-4A50-B499-E03E5C3CC095}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IIS.StaticShim.FunctionalTests", "test\IIS.StaticShim.FunctionalTests\IIS.StaticShim.FunctionalTests.csproj", "{582B07BC-73F4-4689-8557-B039298BD82C}"
```

Comment:
I really don't like the name: StaticShim. BackwardsCompatibility.Tests? RegressionTest?